webapp/common-react/@dbeaver/react-data-grid/src/DataGridHeaderCellContext.ts

```diff
@@ -3,7 +3,7 @@ import type { IGridReactiveValue } from './IGridReactiveValue.js';
 
 export interface IDataGridHeaderCellContext {
   headerElement?: IGridReactiveValue<React.ReactNode, [colIdx: number]>;
-  headerText?: IGridReactiveValue<string, [colIdx: number]>;
+  headerText?: IGridReactiveValue<string | React.ReactNode, [colIdx: number]>;
```

Comment:
`headerText` should be a string, for `React.ReactNode` it's different `headerElement`